dataverse-webapp/src/main/webapp/search-include-fragment.xhtml

```diff
@@ -335,7 +335,7 @@
                 <ui:repeat value="#{SearchIncludeFragment.filterQueries}" var="filterQuery" rendered="#{!SearchIncludeFragment.wasSolrErrorEncountered()}">
                     <ui:param name="friendlyNames" value="#{SearchIncludeFragment.getFriendlyNamesFromFilterQuery(filterQuery)}"/>
                     <h:outputLink value="#{widgetWrapper.wrapURL(page)}" styleClass="facetLink facetSelected" rendered="#{not empty friendlyNames}">
-                        <h:outputText value="#{friendlyNames.get(0)}: #{friendlyNames.get(1)}"/> <span class="glyphicon glyphicon-remove"></span>
+                        <h:outputText value="#{friendlyNames.get(0)}: #{friendlyNames.get(1)}" escape="false"/> <span class="glyphicon glyphicon-remove"></span>
```

Comment:
This could be potentially dangerous: we don't fully control `#{friendlyNames.get(1)}`. What if someone will enter some malicious html in dataset field value? It have potential to be vulnerable to xss attack.